internal/service/encryptionatrestprivateendpoint/model_test.go

```diff
@@ -23,22 +23,22 @@ const (
 )
 
 type sdkToTFModelTestCase struct {
```

Comment:
do we need to update acceptance tests as well to include this data source?

Reply:
yes, somehow lost the adjustment of the acceptance test, thanks!